sys/cddl/compat/opensolaris/sys/vnode.h

```diff
@@ -268,7 +268,7 @@ vn_rename(char *from, char *to, enum uio_seg seg)
 
 	ASSERT(seg == UIO_SYSSPACE);
 
-	return (kern_renameat(curthread, AT_FDCWD, from, AT_FDCWD, to, seg));
+	return (kern_renameat(curthread, AT_FDCWD, cheri_ptr(from,strlen(from)), AT_FDCWD, cheri_ptr(to,strlen(to)), seg));
```

Comment:
I think those bounds are going to be one byte to small.

Reply:
Also a space after the `,` please

Reply:
Fixed!